.github/workflows/test.yml

```diff
@@ -53,6 +53,9 @@ jobs:
           name: coverage-data-${{ matrix.python-version }}
           path: '${{ github.workspace }}/.coverage*'
 
+      - name: Check migrations
+        run: python manage.py makemigrations --check
```

Comment:
You can add this to the test suite as well.

https://adamj.eu/tech/2024/06/23/django-test-pending-migrations/

Reply:
Actually it's already part of the pre-commit hooks so in theory you shouldn't be able to commit code that needs migrations and doesn't include them.

Reply:
Pre-commit should be bypassed with a param on the console if someone didn't install Pre-commit

Reply:
I'd like to address this particular issue outside of precommit. Interesting idea to do it in tests directly...I might go that direction seeing as it's a bit janky to put it in the pypi-test as I've done here (just did it to save time during CI to piggyback on the environment)

Reply:
> Pre-commit should be bypassed with a param on the console, and if someone didn't install Pre-commit

Yes we need pre-commit either running manually as a workflow run, or integrating the bot from pre-commit.ci

I think we've already got PRs from people who don't have pre-commit installed.